content/release/v3/3.4/ReleaseNotes344.md

```diff
@@ -0,0 +1,53 @@
+<!--
+title: "Contrast 3.4.4 - September 2017"
+description: "Contrast 3.4.4 September 2017"
+tags: "3.4.4 September Release Notes"
+-->
+
+Congratulations, Ruby! Version 1.0.0 has arrived along with new bugtracker integration features like configuration for customized URLs and automatic ticket creation. 
+
+## Fixes
+
+* Added event tag and source type options CONTRAST-17145, CONTRAST-17322
```

Comment:
Thought we were going to stop referencing JIRA tickets in the notes. Is that not the case? If it is, let's remove all of the ticket numbers.